src/lib/radio/radio/radio-constants.ts

```diff
@@ -0,0 +1,53 @@
+import { COMPONENT_NAME_PREFIX } from '../../constants';
+import { ARIAAttribute } from '../../core/utils/a11y-utils';
+import { supportsElementInternalsAria } from '../../core/utils/feature-detection';
+
+const elementName: keyof HTMLElementTagNameMap = `${COMPONENT_NAME_PREFIX}radio`;
+
+const attributes = {
+  CHECKED: 'checked',
+  DEFAULT_CHECKED: 'default-checked',
+  VALUE: 'value',
+  DENSE: 'dense',
+  DISABLED: 'disabled',
+  REQUIRED: 'required',
+  READONLY: 'readonly',
+  LABEL_POSITION: 'label-position',
+  TABINDEX: 'tabindex'
+};
+
+const selectors = {
+  ROOT: '.forge-radio',
+  LABEL: '#label',
+  STATE_LAYER: 'forge-state-layer'
+};
+
+const events = {
+  CHANGE: 'change',
+  INPUT: 'input'
+};
+
+export const observedAriaAttributes: ARIAAttribute[] = supportsElementInternalsAria()
+  ? []
+  : [
+    'role',
+    'aria-checked',
+    'aria-disabled',
+    'aria-invalid',
+    'aria-label',
+    'aria-readonly',
+    'aria-required'
+  ];
+
+export const RADIO_CONSTANTS = {
+  elementName,
+  attributes,
+  selectors,
+  events,
+  observedAriaAttributes
+};
+
+export const tryCheck = Symbol('tryCheck');
```

Comment:
Maybe comment what this `Symbol` is used for either in the class implementation or here? Had to look it up when I first saw it just to be sure I was understanding.

Reply:
Will do. I'm also concerned that the name is generic enough that we could run into conflicts in the future. I might change it to something more unique to the radio or move it into the top level constants to allow other components to use it.